<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartDesk</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/res.qrc">
    <normaloff>:/usedimage/package.svg</normaloff>:/usedimage/package.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>221</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="combinedWidget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="1" column="0">
           <widget class="QStackedWidget" name="stackedWidget"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>当前界面</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>16</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menunew">
     <property name="title">
      <string>新建</string>
     </property>
     <addaction name="actiontxt_file"/>
     <addaction name="actionscv_file"/>
    </widget>
    <widget class="QMenu" name="recentFile">
     <property name="title">
      <string>记录</string>
     </property>
    </widget>
    <addaction name="menunew"/>
    <addaction name="recentFile"/>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionclose"/>
    <addaction name="actiondownload"/>
   </widget>
   <widget class="QMenu" name="menuaction">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actiondo"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionshe"/>
    <addaction name="actionfind"/>
    <addaction name="actiontest"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuaction"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen" resource="../../resources/res.qrc">
     <normaloff>:/image/database_go.svg</normaloff>:/image/database_go.svg</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave" resource="../../resources/res.qrc">
     <normaloff>:/image/table_save.svg</normaloff>:/image/table_save.svg</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditClear"/>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actiondownload">
   <property name="icon">
    <iconset theme="emblem-downloads" resource="../../resources/res.qrc">
     <normaloff>:/image/arrow_refresh_small.svg</normaloff>:/image/arrow_refresh_small.svg</iconset>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actiontxt_file">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>txt文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+T</string>
   </property>
  </action>
  <action name="actionscv_file">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>scv文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N, Ctrl+C</string>
   </property>
  </action>
  <action name="actionshe">
   <property name="icon">
    <iconset resource="../../resources/res.qrc">
     <normaloff>:/png/Settings.png</normaloff>:/png/Settings.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::UserAvailable"/>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </action>
  <action name="actiondo">
   <property name="text">
    <string>操纵csv</string>
   </property>
  </action>
  <action name="actionfind">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
